src/styles/blocks/card.scss

```diff
@@ -0,0 +1,119 @@
+.card {
+  width: $card-width;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+
+  border: 1px solid $card-border-color;
+  border-radius: 5px;
+
+  font-size: 12px;
+
+  color: $main-card-color;
+
+  transition:
+    transform 300ms linear,
+    color 300ms linear;
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(120%);
+  }
+
+  &__title {
+    display: block;
+    margin: 0 0 4px;
```

Comment:
`margin-bottom`?

Reply:
@VolodymyrKirichenko I reset margin-top in this line too